06-building-a-program-start-to-finish/writing-tests/Output/inverted-index-spec.js

```diff
@@ -0,0 +1,180 @@
+/* global describe, it, expect */
+const fantasyBook = require('../../src/test books/fantasyBook.json'),
+      quotesAndRap = require('../../src/test books/quotesAndRap.json'),
+      religiousBook = require('../../src/test books/religiousBook.json'),
+      testsBook = require('../../src/test books/testsBook.json'),
+      notArray = require('../../src/test books/notArray.json'),
+      notArrayOfObjects =
+                      require('../../src/test books/notArrayOfObjects.json'),
+      moreThanTwoKeys = require('../../src/test books/moreThanTwoKeys.json'),
+      notAllTextKeys = require('../../src/test books/notAllTextKeys.json'),
+      capitalTitleKey= require('../../src/test books/capitalTitleKey.json'),
+      titleValueAsNumber =
+                        require('../../src/test books/titleValueAsNumber.json'),
+      textValueAsNumber =
+                        require('../../src/test books/textValueAsNumber.json');
+      religiousBookIndexes =
+                      require('../../src/test books/religiousBookIndexes.json');
+
```

Comment:
Ok. I see what you are trying to do here. Titles are still too long and what you are doing by breaking to the next line because you want to avoid violating the 80 max-len rule is a scam. 

Can these test json files be in a one folder which shares a folder with your InvertedIndex.test.js file?

I belivee the names can still be shorter though
sample `religiousBookIndexes ` can be `religiousIndex`
sample `titleValueAsNumber` can be `numberTitle`

Please refactor test w=and we continue from there